lib/Parse/SyntaxParsingContext.cpp

```diff
@@ -163,7 +164,8 @@ const SyntaxParsingContext *SyntaxParsingContext::getRoot() const {
 }
 
 ParsedTokenSyntax SyntaxParsingContext::popToken() {
-  return popIf<ParsedTokenSyntax>().getValue();
+  auto tok = popIf<ParsedTokenSyntax>();
+  return std::move(tok.getValue());
```

Comment:
In https://github.com/apple/swift/pull/27190, this was:
```cpp
ParsedTokenSyntax &&SyntaxParsingContext::popToken() {
  return std::move(popIf<ParsedTokenSyntax>().getValue());
}
```
This returns a reference to the temporary local value of `popIf<ParsedTokenSyntax>()`.

Reply:
@compnerd Could you check if this change doesn't break Windows build?